app/uploaders/disability_claim_document_uploader.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class DisabilityClaimDocumentUploader < CarrierWave::Uploader::Base
+  MAX_FILE_SIZE = 25.megabytes
```

Comment:
So if I recall this 25.megabytes is an NSOC requirement as well, right? Would it be a good idea to build some of these into a BaseUploader that others could inherit from in the future and document accordingly? This is just a thought though, might save someone else some headache in the future.